docs/events/timeline-ui-overview.asciidoc

```diff
@@ -2,84 +2,72 @@
-and share your findings with others, attach your Timeline to a
-<<cases-overview, case>>.
+available for later review. To record and share your findings with others, attach your Timeline to a <<cases-overview, case>>.
 
-TIP: An untitled Timeline is saved as a draft. To attach a Timeline to a
+TIP: Untitled Timelines are saved as drafts. To attach a Timeline to a
 <<cases-overview, case>>, you must give it a title.
 
 In addition to Timelines, you can create and attach Timeline templates to
 <<detection-engine-overview, detection rules>>. Timeline templates allow you to
-define the source event fields used when you investigate a detections alert in
+define the source event fields used when you investigate alerts in
 Timeline. You can select whether the fields use predefined values or values
-retrieved from the alert. For more information, see <<timeline-templates-ui>>.
+retrieved from the alert. For more information, refer to <<timeline-templates-ui>>.
 
 
 [discrete]
 [[refine-timeline-results]]
 == View and refine Timeline results
 
-Using the drop-down options
-by the KQL bar, you can select whether <<det-engine-terminology, alerts>>,
-other raw events, or both are displayed in the Timeline.
+Using the drop-down options by the KQL bar, you can select whether <<det-engine-terminology, alerts>>, other raw events, or both are displayed in the Timeline.
 
 [discrete]
 [[conf-timeline-display]]
 == Configure Timeline event context and display
 
-Many events in Timeline are presented in easy-to-follow rendered views that
-include relevant contextual information. These views are called
-*Event Renderers*, and they can be configured via the Settings icon in the upper
-left corner of the result's pane:
+Many types of events automatically appear in preconfigured views that provide relevant contextual information, called *Event Renderers*. You can see and turn them on or off with the Settings icon in the upper left corner of the results pane:
 
 [role="screenshot"]
 image::images/timeline-ui-renderer.png[]
 
-The example above displays a flow event renderer. If you see a particular item
-that interests you, you can drag it to the dropzone for further investigation.
+The example above displays the Flow event renderer, which highlights the movement of data between its source and destination. If you see a particular part of the rendered event that interests you, you can drag it up to the drop zone below the query bar for further investigation.
 
-Other display options include:
+You can also modify a Timeline's display in other ways:
 
-* View in full screen mode
-* Add, remove, reorder, and resize columns
-* Add notes to individual events
-* Add investigation notes for the entire Timeline
-* Pin interesting events in the Timeline
+* View it in full screen mode.
+* Add, remove, reorder, or resize columns.
+* Add notes to individual events.
+* Add investigation notes to the entire Timeline.
+* Pin interesting events to the Timeline.
```

Comment:
For an unordered list like this, I would remove the `.` from the end as the original had.